ratis-server-api/src/main/java/org/apache/ratis/statemachine/StateMachine.java

```diff
@@ -207,7 +208,7 @@ interface LeaderEventApi {
      *
      * @see org.apache.ratis.server.RaftServerConfigKeys.Rpc#SLOWNESS_TIMEOUT_KEY
      */
-    default void notifyFollowerSlowness(RoleInfoProto roleInfoProto) {}
```

Comment:
We should first deprecate the API and remove it in a later version.

Reply:
Ah, Thanks. I missed that.